packages/custom-esbuild/README.md

```diff
@@ -100,7 +100,7 @@ Builder options:
   "build": {
     "builder": "@angular-builders/custom-esbuild:application",
     "options": {
-      "plugins": ["./esbuild/plugins.ts", "./esbuild/plugin-2.js"],
+      "plugins": ["./esbuild/plugins.ts", { "path": "./esbuild/plugin-2.js", "options": { "key": "value" } }],
```

Comment:
Could we please have a more practical example demonstrating its usefulness? For instance, could you provide an example involving a real plugin that necessitates options, or perhaps share a scenario from your own application illustrating why it's essential?

Reply:
makes sense for me, but I found a bug which makes using 3rd party plugins (almost) impossible.

Paths are always joined with the workspaceRoot `loadModule<Plugin | Plugin[]>(path.join(workspaceRoot, pluginPath), tsConfig, logger)`. This makes it impossible to load modules by the package name.

I would propose to remove this and just use the plugin path without the workspace root. Unless I missed something, this would be inline with the behavior of custom-webpack.
If you agree, I can do this in another MR.

Reply:
@spike-rabbit Hmmm I wonder if that's the reason for `JSON.stringify` (CI fails now):  
![image](https://github.com/just-jeb/angular-builders/assets/9823087/29343559-3c08-4364-9834-206c746de059)

Reply:
- JSON stringify is back
- updated the README